packages/build/tests/core/snapshots/tests.js.md

```diff
@@ -973,6 +974,162 @@ Generated by [AVA](https://ava.li).
+    ␊
+    ────────────────────────────────────────────────────────────────␊
+      Netlify Build Complete                                        ␊
+    ────────────────────────────────────────────────────────────────␊
+    ␊
+    (Netlify Build completed in 1ms)`
+
+## Functions config is passed to zip-it-and-ship-it (2)
+
+> Snapshot 1
+
+    `␊
+    ────────────────────────────────────────────────────────────────␊
+      Netlify Build                                                 ␊
+    ────────────────────────────────────────────────────────────────␊
+    ␊
+    > Version␊
+      @netlify/build 1.0.0␊
+    ␊
+    > Flags␊
+      debug: true␊
+      featureFlags: {}␊
+      repositoryRoot: /file/path␊
+      testOpts:␊
+        pluginsListUrl: test␊
+        silentLingeringProcesses: true␊
+    ␊
+    > Current directory␊
+      /file/path␊
+    ␊
+    > Config file␊
+      /file/path␊
+    ␊
+    > Resolved config␊
+      build:␊
+        functions: /file/path␊
+    ␊
+    > Context␊
+      production␊
+    ␊
+    ────────────────────────────────────────────────────────────────␊
+      1. Functions bundling                                         ␊
+    ────────────────────────────────────────────────────────────────␊
+    ␊
+    Packaging Functions from functions directory:␊
+     - function_one.js␊
+    ␊
+    ␊
+    (Functions bundling completed in 1ms)␊
+    ␊
+    ────────────────────────────────────────────────────────────────␊
+      Netlify Build Complete                                        ␊
+    ────────────────────────────────────────────────────────────────␊
+    ␊
+    (Netlify Build completed in 1ms)`
+
+## Functions config is passed to zip-it-and-ship-it (3)
+
+> Snapshot 1
+
+    `␊
+    ────────────────────────────────────────────────────────────────␊
+      Netlify Build                                                 ␊
+    ────────────────────────────────────────────────────────────────␊
+    ␊
+    > Version␊
+      @netlify/build 1.0.0␊
+    ␊
+    > Flags␊
+      debug: true␊
+      featureFlags: {}␊
+      repositoryRoot: /file/path␊
+      testOpts:␊
+        pluginsListUrl: test␊
+        silentLingeringProcesses: true␊
+    ␊
+    > Current directory␊
+      /file/path␊
+    ␊
+    > Config file␊
+      /file/path␊
+    ␊
+    > Resolved config␊
+      build:␊
+        functions: /file/path␊
+    ␊
+    > Context␊
+      production␊
+    ␊
+    ────────────────────────────────────────────────────────────────␊
+      1. Functions bundling                                         ␊
+    ────────────────────────────────────────────────────────────────␊
+    ␊
+    Packaging Functions from functions directory:␊
+     - function_one.js␊
+     - function_two.js␊
+    ␊
+    ␊
+    ────────────────────────────────────────────────────────────────␊
+      Dependencies installation error                               ␊
+    ────────────────────────────────────────────────────────────────␊
+    ␊
+      Error message␊
```

Comment:
This failure is expected because the config in this fixture is flagging as external two out of the three modules. The one that is left out of that list causes the build to fail, because the module doesn't exist.